model/authentication.go

```diff
@@ -74,6 +75,16 @@ func (auth *Authentication) ToResponse() *AuthenticationResponse {
 
 func (auth *Authentication) ToInternalResponse() *AuthenticationInternalResponse {
 	resourceID := strconv.FormatInt(auth.ResourceID, 10)
+	var pass string
+	if auth.Password == nil {
+		pass = ""
+	} else {
```

Comment:
Since the `AuthenticationInternalResponse` has an `omitempty` tag, maybe we don't need to default it to the empty string

Reply:
That is true, I let it default to `""` since that is the string default value.